scripts/build.py

```diff
@@ -130,11 +131,13 @@ def build_files(source, target, ignore):
                 continue
             target_path = source_path.replace(source, target)
 
-            # Only copy files in ignore or with extensions mentioned in
-            # FILE_EXTENSIONS_TO_COPY_WITHOUT_MINIFICATION variable.
-            file_patterns_to_ignore = (
-                ignore + FILE_EXTENSIONS_TO_COPY_WITHOUT_MINIFICATION)
-            if any(p in source_path for p in file_patterns_to_ignore):
+            # Ignore files with certain extensions
+            if any(p in source_path for p in FILE_EXTENSIONS_TO_IGNORE):
```

Comment:
source_path.endswith(p) is stricter and more in line with what you want to check, I think?